mautic/mautic
# added timeline event when a contact is eddited from the contact-profi…

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | Enhancement
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/106
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2471
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds timeline event when a contact is moved to a stage from the contact's timeline, link to stage from timeline and reference to stage on the changelog table

#### Steps to test this PR:
1. Apply PR
2. Go to a contact, change the contact's stage, a new entry on the timeline should appear.
